languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/.docs/hints.md

```diff
@@ -0,0 +1,28 @@
+## 1. Make `preparation-time-in-minutes` easier to use
+
+    - a "rest parameter" will collect all arguments not consumed by other parameters into a list.
```

Comment:
A real nitpick, it might be nice to have all of these list-items capitalised like sentences :slightly_smiling_face: